mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksFactory.kt

```diff
@@ -4,8 +4,7 @@ import androidx.annotation.RestrictTo
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 
-@RestrictTo(RestrictTo.Scope.LIBRARY)
-class MavericksFactory<VM : MavericksViewModel<S>, S : MavericksState>(
+internal class MavericksFactory<VM : MavericksViewModel<S>, S : MavericksState>(
```

Comment:
we had been using this at airbnb. we may not need to any longer with the upstreamed mocking, but i'd rather leave this public for now just in case we need it. this doesn't seem like one that will accidentally be used

Reply:
@elihart It's less about using it and more about it showing up every tine you type Mavericks in intellisense for things like your ViewModel/State/etc.

Reply:
For that reason, I want to keep the `Mavericks*` public class namespace as small as possible.